asteroid/masknn/convolutional.py

```diff
@@ -383,6 +388,112 @@ def get_config(self):
+        in_chan (int): Number of input channels.
+        out_chan (int): Number of output channels.
+        kernel_size (Tuple[int, int]): Convolution kernel size.
+        stride (Tuple[int, int]): Convolution stride.
+        padding (Tuple[int, int]): Convolution padding.
+        norm_type (str, optional): Type of normalization to use.
+            See ``asteroid.masknn.norms`` for valid values.
+        activation (str, optional): Type of activation to use.
+            See ``asteroid.masknn.activations`` for valid values.
+
+    References:
+        [1] : "Phase-aware Speech Enhancement with Deep Complex U-Net",
+        Hyeong-Seok Choi et al.
+        https://arxiv.org/abs/1903.03107
+    """
+
+    def __init__(
+        self,
+        in_chan,
+        out_chan,
+        kernel_size,
+        stride,
+        padding,
+        norm_type="bN",
+        activation="leaky_relu",
+    ):
+        super().__init__()
+
+        self.conv = complex_nn.ComplexConv2d(in_chan, out_chan, kernel_size, stride, padding)
+
+        self.norm = norms.get_complex(norm_type)(out_chan)
+
+        activation_class = activations.get_complex(activation)
+        self.activation = activation_class()
+
+    def forward(self, x: complex_nn.ComplexTensor):
+        return self.activation(self.norm(self.conv(x)))
+
+
+class DCUNetComplexDecoderBlock(nn.Module):
+    """Decoder block as proposed in [1].
+
+    Args:
+        in_chan (int): Number of input channels.
+        out_chan (int): Number of output channels.
+        kernel_size (Tuple[int, int]): Convolution kernel size.
+        stride (Tuple[int, int]): Convolution stride.
+        padding (Tuple[int, int]): Convolution padding.
+        norm_type (str, optional): Type of normalization to use.
+            See ``asteroid.masknn.norms`` for valid values.
+        activation (str, optional): Type of activation to use.
+            See ``asteroid.masknn.activations`` for valid values.
+
+    References:
+        [1] : "Phase-aware Speech Enhancement with Deep Complex U-Net",
+        Hyeong-Seok Choi et al.
+        https://arxiv.org/abs/1903.03107
+    """
+
+    def __init__(
+        self,
+        in_chan,
+        out_chan,
+        kernel_size,
+        stride,
+        padding,
+        norm_type="bN",
+        activation="leaky_relu",
+    ):
+        super().__init__()
+
+        self.in_chan = in_chan
+        self.out_chan = out_chan
+        self.kernel_size = kernel_size
+        self.stride = stride
+        self.padding = padding
+
+        self.deconv = complex_nn.ComplexConvTranspose2d(
+            in_chan, out_chan, kernel_size, stride, padding
+        )
+
+        self.norm = norms.get_complex(norm_type)(out_chan)
+
+        activation_class = activations.get_complex(activation)
+        self.activation = activation_class()
+
+    def forward(self, x: complex_nn.ComplexTensor):
+        return self.activation(self.norm(self.deconv(x)))
+
+
+class DCUMaskNet(BaseDCUMaskNet):
```

Comment:
Maybe document here that there is the `default_architecture` classmethod? With valid architecture names? 
Or add the `show_available_architectures` class method in the base class? 
Otherwise, I don't see how the user discovers which architecture to use.

Reply:
This is now documented here.